backend/src/permissions/permissions.service.spec.ts

```diff
@@ -169,6 +171,23 @@ describe('PermissionsService', () => {
     eventEmitter = module.get<EventEmitter2>(EventEmitter2);
   });
 
+  function mockMayReadTrue(): void {
+    jest.spyOn(service, 'mayRead').mockImplementationOnce(async () => {
+      return true;
+    });
+  }
+
+  function mockMayWriteTrue(): void {
+    jest.spyOn(service, 'mayWrite').mockImplementationOnce(async () => {
+      return true;
+    });
+  }
+
+  function mockIsOwner(yes: boolean): void {
```

Comment:
Well. either you change the type of `yes` to `true` instead of boolean..... or you rename the parameter 😁

Reply:
changed